includes/admin/views/omise-page-settings.php

```diff
@@ -98,85 +98,91 @@
-		<table class="form-table">
-			<tbody>
-				<tr>
-					<th scope="row"><label for="sandbox"><?php _e( 'Available Payment Methods', 'omise' ); ?></label></th>
-					<td>
-						<table class="widefat fixed striped" cellspacing="0">
-							<thead>
-								<tr>
-									<?php
-										$columns = array(
-											'name'    => __( 'Payment Method', 'omise' ),
-											'status'  => __( 'Enabled', 'omise' ),
-											'setting' => ''
-										);
+		<?php if ($settings['account_country']) : ?>
+			<p><?php _e( 'The table below is a list of available payment methods that you can enable in your WooCommerce store.', 'omise' ); ?></p>
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th scope="row"><label for="sandbox"><?php _e( 'Available Payment Methods', 'omise' ); ?></label></th>
+						<td>
+							<table class="widefat fixed striped" cellspacing="0">
+								<thead>
+									<tr>
+										<?php
+											$columns = array(
+												'name'    => __( 'Payment Method', 'omise' ),
+												'status'  => __( 'Enabled', 'omise' ),
+												'setting' => ''
+											);
 
-										foreach ( $columns as $key => $column ) {
-											switch ( $key ) {
-												case 'status' :
-												case 'setting' :
-													echo '<th style="text-align: center; padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
-													break;
+											foreach ( $columns as $key => $column ) {
+												switch ( $key ) {
+													case 'status' :
+													case 'setting' :
+														echo '<th style="text-align: center; padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
+														break;
 
-												default:
-													echo '<th style="padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
-													break;
-											}
+													default:
+														echo '<th style="padding: 10px;" class="' . esc_attr( $key ) . '">' . esc_html( $column ) . '</th>';
+														break;
+												}
 
-										}
-									?>
-								</tr>
-							</thead>
-							<tbody>
-								<?php
-								$available_gateways = array(
-									new Omise_Payment_Alipay,
-									new Omise_Payment_Billpayment_Tesco,
-									new Omise_Payment_Creditcard,
-									new Omise_Payment_Installment,
-									new Omise_Payment_Internetbanking,
-									new Omise_Payment_Truemoney
-								);
-								foreach ( $available_gateways as $gateway ) :
+											}
+										?>
+									</tr>
+								</thead>
+								<tbody>
+									<?php
+									$available_gateways = array(
+										new Omise_Payment_Alipay,
+										new Omise_Payment_Billpayment_Tesco,
+										new Omise_Payment_Creditcard,
+										new Omise_Payment_Installment,
+										new Omise_Payment_Internetbanking,
+										new Omise_Payment_Truemoney
```

Comment:
Wow - they're even alphabetised! :)

Reply:
I do have the feeling this list of available gateways really belongs at a higher level as it feels like something that may be used again elsewhere. It's ok here for now though I guess

Reply:
Thanks for the input 👍 
Now you have mentioned and it got me to think that probably the main `Omise` class (omise-woocommerce.php) would also likely to be a place where this can also be implemented if not too complex.

Note for idea (but not in anytime soon)
```php
foreach ( Omise::available_payment_methods() as $methods ) {
    // ...
}
```

Reply:
also note for myself. I think `gateways` is a bad name. Should be `methods`

Reply:
Yep. I mentioned it because I'm doing something similar in my Prestashop refactoring